OpenSim/Simulation/Model/Geometry.h

```diff
@@ -232,6 +228,7 @@ class OSIMSIMULATION_API LineGeometry : public Geometry
         upd_end_point() = aPoint2;
     }
 
+protected:
     /** Virtual method to map LineGeometry to SimTK::Array of
     SimTK::DecorativeGeometry.  Appearance, Transforms are handled by base
     Geometry class. */
```

Comment:
In the context of this method's comment, it sounds like `LineGeometry::createDecorativeGeometry()` must call `Super::createDecorativeGeometry()` to handle Appearance and Transforms. What do you mean by handle them?